.github/workflows/CICD.yml

```diff
@@ -178,29 +182,50 @@ jobs:
           extensions: ${{ env.extensions }}
           coverage: none
 
-      - name: Install Composer dependencies
-        uses: ramsey/composer-install@57532f8be5bda426838819c5ee9afb8af389d51a # 3.0.0
-        with:
-          composer-options: --no-dev
-
       - name: Use Node.js 20
         uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af # v4.1.0
         with:
             node-version: 20
 
-      - name: Install
-        run: npm ci
```

Comment:
IIRC we only just changed to `ci`. It looks like this block was always redundant (handled by the `make` below) but the `make` uses `npm install`. Do we want this or should we change the Makefile too?

Reply:
I think we should change the makefile too.